networking/k8s_nmstate/k8s-nmstate-using-kubernetes-nmstate.adoc

```diff
@@ -0,0 +1,20 @@
+[id="k8s-nmstate-using-kubernetes-nmstate"]
+
+= Using Kubernetes NMState
+include::modules/common-attributes.adoc[]
+:context: k8s-nmstate-using
+
+The `kubernetes-nmstate` operator provides a Kubernetes API for peforming state-driven network configuration across the {product-title} cluster's nodes with `nmstate`. `kubernetes-nmstate` provides users with functionality to configure various network interface types, DNS and routing on cluster nodes. Additionally, the daemons on the cluster nodes periodically report on the state of each node's network interfaces to the API server.
+
+
+[NOTE]
```

Comment:
There is a module called technology-preview.adoc that you could use here instead